assets/ics/irc-meeting.ics

```diff
@@ -0,0 +1,180 @@
+BEGIN:VCALENDAR
+PRODID:-//Google Inc//Google Calendar 70.9054//EN
+VERSION:2.0
+CALSCALE:GREGORIAN
+METHOD:PUBLISH
+X-WR-CALNAME:Bitcoin-dev
+X-WR-TIMEZONE:Atlantic/Reykjavik
+X-WR-CALDESC:Bitcoin development meetings
+BEGIN:VTIMEZONE
+TZID:Atlantic/Reykjavik
+X-LIC-LOCATION:Atlantic/Reykjavik
+BEGIN:STANDARD
+TZOFFSETFROM:+0000
+TZOFFSETTO:+0000
+TZNAME:GMT
+DTSTART:19700101T000000
+END:STANDARD
+END:VTIMEZONE
+BEGIN:VEVENT
+DTSTART:20200811T150000Z
+DTEND:20200811T160000Z
+RRULE:FREQ=WEEKLY;WKST=MO;UNTIL=20210322T235959Z;INTERVAL=2;BYDAY=TU
+EXDATE:20201229T150000Z
+DTSTAMP:20250403T004023Z
+UID:6j5og22fjsf55qlecj5d5lslik@google.com
+CREATED:20200806T195438Z
+LAST-MODIFIED:20210303T195953Z
+SEQUENCE:2
+STATUS:CONFIRMED
+SUMMARY:Bitcoin Core P2P meeting
+TRANSP:OPAQUE
+END:VEVENT
+BEGIN:VEVENT
+DTSTART:20210223T210000Z
+DTEND:20210223T220000Z
+DTSTAMP:20250403T004023Z
+UID:6j5og22fjsf55qlecj5d5lslik@google.com
+RECURRENCE-ID:20210223T150000Z
+CREATED:20200806T195438Z
+LAST-MODIFIED:20210303T195953Z
+SEQUENCE:3
+STATUS:CONFIRMED
+SUMMARY:Bitcoin Core P2P meeting
+TRANSP:OPAQUE
+END:VEVENT
+BEGIN:VEVENT
+DTSTART:20210309T210000Z
+DTEND:20210309T220000Z
+DTSTAMP:20250403T004023Z
+UID:6j5og22fjsf55qlecj5d5lslik@google.com
+RECURRENCE-ID:20210309T150000Z
+CREATED:20200806T195438Z
+LAST-MODIFIED:20210308T195859Z
+SEQUENCE:3
+STATUS:CONFIRMED
+SUMMARY:Bitcoin Core P2P meeting
+TRANSP:OPAQUE
+END:VEVENT
+BEGIN:VEVENT
+DTSTART:20210323T210000Z
+DTEND:20210323T220000Z
+RRULE:FREQ=WEEKLY;WKST=MO;UNTIL=20210726T235959Z;INTERVAL=2;BYDAY=TU
+DTSTAMP:20250403T004023Z
+UID:p90eth8a3d6gsn9nqrodlar405@google.com
+CREATED:20200806T195438Z
+LAST-MODIFIED:20210726T093115Z
+SEQUENCE:3
+STATUS:CONFIRMED
+SUMMARY:Bitcoin Core P2P meeting
+TRANSP:OPAQUE
+END:VEVENT
+BEGIN:VEVENT
+DTSTART;TZID=Atlantic/Reykjavik:20150924T190000
+DTEND;TZID=Atlantic/Reykjavik:20150924T200000
+RRULE:FREQ=WEEKLY;UNTIL=20230524T235959Z;BYDAY=TH
+EXDATE;TZID=Atlantic/Reykjavik:20201224T190000
+EXDATE;TZID=Atlantic/Reykjavik:20201231T190000
+DTSTAMP:20250403T004023Z
+UID:ho8ddsk5g3c26838p47hh6m2bs@google.com
+CREATED:20150922T141502Z
+LAST-MODIFIED:20230504T203810Z
+LOCATION:#bitcoin-dev on Freenode IRC

```

Comment:
Still says Freenode, also definitely not been in the #bitcoin-dev channel for almost a decade.

Reply:
This is for the older recurring event, but I should fix the channel in the currently active event below.